openssl/src/md.rs

```diff
@@ -30,52 +27,52 @@ cfg_if! {
                 }
             }
         }
+    }
+}
 
-        impl ForeignType for Md {
-            type CType = ffi::EVP_MD;
-            type Ref = MdRef;
+impl ForeignType for Md {
+    type CType = ffi::EVP_MD;
+    type Ref = MdRef;
 
-            #[inline]
-            unsafe fn from_ptr(ptr: *mut Self::CType) -> Self {
-                Md(ptr)
-            }
+    #[inline]
+    unsafe fn from_ptr(ptr: *mut Self::CType) -> Self {
+        Md(ptr)
+    }
 
-            #[inline]
-            fn as_ptr(&self) -> *mut Self::CType {
-                self.0
-            }
-        }
+    #[inline]
+    fn as_ptr(&self) -> *mut Self::CType {
+        self.0
+    }
+}
 
-        impl Deref for Md {
-            type Target = MdRef;
+impl Deref for Md {
+    type Target = MdRef;
 
-            #[inline]
-            fn deref(&self) -> &Self::Target {
-                unsafe {
-                    MdRef::from_ptr(self.as_ptr())
-                }
-            }
-        }
+    #[inline]
+    fn deref(&self) -> &Self::Target {
+        unsafe { MdRef::from_ptr(self.as_ptr()) }
+    }
+}
 
-        impl DerefMut for Md {
-            #[inline]
-            fn deref_mut(&mut self) -> &mut Self::Target {
-                unsafe {
-                    MdRef::from_ptr_mut(self.as_ptr())
-                }
-            }
-        }
-    } else {
-        enum Inner {}
```

Comment:
I'm a bit suspicious that this is a safe change. I vaguely remember that the new OpenSSL 3 dynamic lookup APIs were the reason I had to make the new types in the first place.